pop_tools/grid.py

```diff
@@ -320,6 +361,16 @@ def get_grid(grid_name, scrip=False):
     return dso
 
 
+@jit(nopython=True, parallel=True)
+def _generate_KMU(KMT):
+    """Computes KMU from KMT."""
+    KMU = np.zeros_like(KMT)
+    for i in prange(KMT.shape[0]):
+        for j in prange(KMT.shape[1]):
+            KMU[i, j] = min(KMT[i, j], KMT[i - 1, j], KMT[i, j - 1], KMT[i - 1, j - 1])
```

Comment:
You may want to use `np.min()` here. Python's builtin `min()` method doesn't appear on the list of array operations that can be parallelized by numba

See: https://numba.pydata.org/numba-doc/latest/user/parallel.html#supported-operations

Reply:
Thinking more about this, `np.min()` is probably not needed since you are operating on scalars in `min(KMT[i, j], KMT[i - 1, j], KMT[i, j - 1], KMT[i - 1, j - 1])`

Reply:
So keep it as is? My thinking is the loops are parallelized with `prange` and `numba` such that min is just comparing 4 scalars, but in parallel at the `(i, j)` level.